namespaces/docker.yml

```diff
@@ -9,3 +9,103 @@ namespace:
     type: string
     description: >
       Docker container id
+  - name: event
+    type: group
+    description: >
+      Event emitted by docker daemon that describes some activity that occurred in the past
+    fields:
+    - name: action
+      type: string
+      example: create
+      description: >
+        What activity occurred
+    - name: status
+      type: string
+      example: create
+      description: >
+        What is the result of the activity
+    - name: type
+      type: string
+      example: container
+      description: >
+        Container or image or volume or network or daemon or plugin or service or node or secret or config
+    - name: time
+      type: date
+      example: 2017-08-22T14:47:29.146379+00:00
+      description: >
+        Date of the occurrence of the event
+    - name: container
+      type: group
+      description: >
+        What resource the activity targeted
+      fields:
+      - name: name
+        type: string
+        example: my-fedora
+        description: >
+          Name of the container
+      - name: image
+        type: string
+        example: registry.fedoraproject.org/fedora:latest
+        description: >
+          Image from which the container is derived
+      - name: state
+        type: group
+        description: >
+          State of the container
+        fields:
+        - name: Status
```

Comment:
looks like the convention in `go` land is to use fields which begin with an uppercase letter in many places, `go` templates, among other places.  IMO, we should make all of these lower case when storing in logging - we will confuse admins if we require them to use uppercase in some places and lowercase in others.
On the other hand, we are going to be causing confusion for docker administrators . . .